src/pal/src/thread/process.cpp

```diff
@@ -139,6 +139,11 @@ Volatile<LONG> terminator = 0;
 DWORD gPID = (DWORD) -1;
 DWORD gSID = (DWORD) -1;
 
+// Application group ID for this process
+LPCSTR gApplicationGroupId = nullptr;
+int gApplicationGroupIdLength = 0;
+PathCharString gApplicationContainerPath;
```

Comment:
Statics with destructors are problematic. On Unix, the destructors may be running while some other threads are using the static. It results into very hard to repro crashes during shutdown.

Reply:
@jkotas are we allowed to make these pointers,  new them up at the static initialize function and delete them during static cleanup?  I assume that should solve the issue